src/util/FileMonitor.h

```diff
@@ -37,38 +36,36 @@ namespace lsst::qserv::util {
 /// events wait for.
 class FileMonitor {
 public:
-	using Ptr = std::shared_ptr<FileMonitor>;
+    using Ptr = std::shared_ptr<FileMonitor>;
 
-	FileMonitor() = delete;
-	FileMonitor(std::string const& fileName) : _fileName(fileName) {
-		_setup();
-	}
+    FileMonitor() = delete;
+    FileMonitor(std::string const& fileName) : _fileName(fileName) { _setup(); }
 
-	/// Stops and joins _thrd, if joinable.
-	~FileMonitor();
+    /// Stops and joins _thrd, if joinable.
+    ~FileMonitor();
 
-	FileMonitor(FileMonitor const&) = delete;
-	FileMonitor& operator=(FileMonitor const&) = delete;
+    FileMonitor(FileMonitor const&) = delete;
+    FileMonitor& operator=(FileMonitor const&) = delete;
 
-	/// start running `_thrd` containing `checkLoop()`.
-	void run();
+    /// start running `_thrd` containing `checkLoop()`.
+    void run();
 
-	/// Stop the thread `_thrd`.
-	void stop() { _loop = false; }
+    /// Stop the thread `_thrd`.
+    void stop() { _loop = false; }
```

Comment:
You don't need to have this method because it has no users but the class's destructor.